test/integration-tests/shared/computeUniswapV2PairHash.ts

```diff
@@ -0,0 +1,8 @@
+import {calculateInitCodeHash} from "./deployUniswapV2"
```

Comment:
If you don't use the result in the code, I believe it makes sense to remove this file then